honeybee_energy/simulationparameter.py

```diff
@@ -159,11 +159,8 @@ def global_geometry_rules(self):
              Counterclockwise,         !- vertex entry direction
              Relative;                 !- coordinate system
         """
-        values = ('UpperLeftCorner',
-                  'Counterclockwise',
-                  'Relative')
-        comments = ('starting vertex position',
-                    'vertex entry direction',
+        values = ('UpperLeftCorner', 'Counterclockwise', 'Relative')
```

Comment:
This comment is not related to this line but the name of the module. Should we start break down the name of modules with long names. For instance call this one `simulation_parameter`? I know it is not best practice but it is valid and it is much easier to read.

This is one of those topics that we should discuss before the public release since it will create breaking changes.

Reply:
So there is a `simulation` module within honeybee and so I could just put this file within that module and call it `parameter`. So you access it by typing `honeybee.simulation.parameter`. But this SimulationParameter class is the parent class to all of the other sub-modules and classes in the `simulation` module. So I moved it out of the module to distinguish it. Do you think this was a mistake or do you know of any best practice here?  I could really go either way on it so I'll implement whatever you think is best.

Reply:
I'm realizing that I probably should put it in the simulation module, even if there's no way to distinguish it as the parent object when it's in there. Let me know if you agree.

Reply:
Ok. I implemented the name change of `simulation.parameter`.

For my thoughts on the underscores in the module names, I think your talking about a monumental effort across the libraries that will break almost everything (nearly all of the libraries I have worked on have several of these combined-name modules). If we're going to do this, it's better to do it now rather than later when we'd have to put out a bunch of deprecation warnings about it.  Personally, I don't think the change is worth the week or so it would take me to follow it through all of the libraries and Grasshopper plugins. And, as you say, it's not even considered the best practice. So I'm not very inclined to change it.

Reply:
This is not what I was thinking but it doesn't look bad to me.

You are right that using underscore will break everything. If we think it is a good idea with might want to do it closer to the release when everything else is in place but would you be fine doing it for the newer modules which we might add from now on?

Reply:
I think we should have a longer conversation about this. I'd much rather keep things consistent  for now and I'd definitely want to understand your reasoning before I sink a lot of time into addressing this.